NuGet.Config

```diff
@@ -4,5 +4,6 @@
     <add key="AspNetVNext" value="https://www.myget.org/F/aspnetvnext/api/v2" />
     <add key="NuGet" value="https://nuget.org/api/v2/" />
     <add key="AzureAD" value="http://www.myget.org/F/azureadwebstacknightly"/>
+    <add key="aspnet-contrib" value="https://www.myget.org/F/aspnet-contrib"/>
```

Comment:
@haok can you merge this? I'm going offline.
________________________________
From: Kévin Chalet<mailto:notifications@github.com>
Sent: ‎6/‎26/‎2015 4:10 PM
To: aspnet/Security<mailto:Security@noreply.github.com>
Cc: Chris R<mailto:Tratcher@Outlook.com>
Subject: Re: [Security] Replace INonceCache by IDistributedCache (#293)

> @@ -4,5 +4,6 @@
>      <add key="AspNetVNext" value="https://www.myget.org/F/aspnetvnext/api/v2" />
>      <add key="NuGet" value="https://nuget.org/api/v2/" />
>      <add key="AzureAD" value="http://www.myget.org/F/azureadwebstacknightly"/>
> +    <add key="aspnet-contrib" value="https://www.myget.org/F/aspnet-contrib"/>

Ah, that was not clear :smile:

Changes reverted.

---
Reply to this email directly or view it on GitHub:
https://github.com/aspnet/Security/pull/293/files#r33403824